module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -206,21 +212,29 @@ namespace module::input {
                         gyro += s->gyroscope.cast<double>();
                         // Make sure we have servo positions
                         for (uint32_t id = 0; id < 20; ++id) {
-                            auto& original = utility::platform::getRawServo(id, *s);
-                            // Add the sensor values to the system properly
-                            filtered_sensors.servo.push_back({0,
-                                                              id,
-                                                              original.torque_enabled,
-                                                              original.p_gain,
-                                                              original.i_gain,
-                                                              original.d_gain,
-                                                              original.goal_position,
-                                                              original.moving_speed,
-                                                              original.present_position,
-                                                              original.present_speed,
-                                                              original.load,
-                                                              original.voltage,
-                                                              static_cast<float>(original.temperature)});
+                            auto& original    = utility::platform::getRawServo(id, *s);
+                            const auto& error = original.error_flags;
+
+                            // Check for an error on the servo and report it
+                            if (error != RawSensors::Error::OK) {
+                                NUClear::log<NUClear::WARN>(servoErrorString(id, original, error));
+                            }
```

Comment:
I think you will need to push the bad servo packet in this case as well, pretty sure [this function](https://github.com/NUbots/NUbots/blob/c16974f0ad9b4b10545b56d24fff63b6f7d6e569/module/input/SensorFilter/src/SensorFilter.cpp#L241) is expecting a full set of servo readings. Something similar to what we are [doing here](https://github.com/NUbots/NUbots/blob/c16974f0ad9b4b10545b56d24fff63b6f7d6e569/module/input/SensorFilter/src/SensorFilter.cpp#L387-L430).